src/basic/lines.js

```diff
@@ -14,19 +14,30 @@
 	EXTERNAL DEPENDENCIES
  */
 
-import {curry, compose, not, equals, identity, cond} from 'ramda';
+const ramda = require('ramda');
```

Comment:
Ok! But you still can use decomposition for having a cleaner code, right?

What I did was run webpack for tangram and then browserify can detect changes in the bundle so you can use those awesome new features with all the development speed, but with two proceses.

The tangram bundle was optimized to minimize the size using tree shake